shell/platform/linux/fl_text_input_plugin.cc

```diff
@@ -315,6 +315,11 @@ FlTextInputPlugin* fl_text_input_plugin_new(FlBinaryMessenger* messenger) {
   return self;
 }
 
+void fl_text_input_plugin_set_window(FlTextInputPlugin* self,
```

Comment:
This should be set in `fl_text_input_plugin_new` in the same way as done for `fl_mouse_cursor_plugin_new` - the window never changes.

Reply:
Done